android/src/main/kotlin/com/gojuno/commander/android/Adb.kt

```diff
@@ -22,6 +22,11 @@ private val buildTools: String? by lazy {
 }
 val aapt: String by lazy { buildTools?.let { "$buildTools/aapt" } ?: "" }
 
+fun deviceModel(serial: String): Observable<String> = process(listOf(adb, "-s", serial, "shell", "getprop ro.product.model undefined"))
```

Comment:
Let's use Single here, it's not gonna change nor we actually observe it :)

Reply:
We also need to test it (parsing, trimming)

Reply:
And looks like we can pass AdbDevice as parameter rather than just a string that could allow someone to pass wrong value without realizing it